interactive_engine/executor/ir/graph_proxy/src/adapters/gs_store/read_graph.rs

```diff
@@ -472,6 +486,17 @@ where
     Vertex::new(id, Some(label), DynDetails::lazy(details))
 }
 
+#[inline]
+fn to_hybrid_runtime_vertex<V>(v: V) -> Vertex
```

Comment:
Btw, could you please add similar logic for edges (i.e., use `LazyEdgeDetails` and `HybridEdgeDetails`)? For now, the logic is `get_all_props` for edges, and then trim it, which is not a good practice.